conf.py

```diff
@@ -0,0 +1,107 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import os.path
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'NeuroPoly'
+copyright = '2021, NeuroPoly'
+author = 'NeuroPoly'
+
+
+# -- General configuration ---------------------------------------------------
+
+root_doc = 'README'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+        'myst_parser', # there's also myst_nb, which supports embedding Jupyter notebooks, but is heavier.
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ['_build', '.venv', 'Thumbs.db', '.DS_Store']
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'sphinx_book_theme'
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
+html_css_files = ["theme.css"]
+
+html_title = "NeuroPoly Lab"
```

Comment:
This is personal preference, but this text looks a little redundant next to the logo:

![Screenshot from 2021-11-03 10-54-13](https://user-images.githubusercontent.com/16181459/140084391-88827486-4ffe-48f8-9c28-e52039e9a672.png)

Can we consider setting [`logo_only`](https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html#confval-logo_only)? This is also what SCT does: https://spinalcordtoolbox.com/

Reply:
I had this set

https://github.com/neuropoly/neuropoly-docs/blob/554f2fb765ae7526d57095bca678d331992ef70e/conf.py#L75

but I hesitated because the logo isn't accessible. It looks good on Firefox:

![2021-11-03-121029_274x252_scrot](https://user-images.githubusercontent.com/987487/140098897-558ca9b1-6618-4c82-a069-a2b3e5e8f984.png)

but if you open it in `lynx` it just says "logo":

![2021-11-03-121024_349x223_scrot](https://user-images.githubusercontent.com/987487/140098898-585354da-446b-4516-a508-ae7ca22bef83.png)

This theme generated

```html
<a class="navbar-brand text-wrap" href="../index.html">
      
        <!-- `logo` is deprecated in Sphinx 4.0, so remove this when we stop supporting 3 -->
        
      
      
      <img src="../_static/logo.png" class="logo" alt="logo">
      
      
    </a>
```

For you'd want

```html
<a class="navbar-brand text-wrap" href="../index.html">
      
        <!-- `logo` is deprecated in Sphinx 4.0, so remove this when we stop supporting 3 -->
        
      
      
      <img src="../_static/logo.png" class="logo" alt="NeuroPoly Lab logo">
      
      
    </a>
```

I wonder if there is a setting to control what goes in `alt=`? We can add a file to `_templates/` to tweak it too, but that might be a bit heavy-handed. I dunno. Thoughts?

Reply:
I'll do what you ask for now and we can fix it later -> #26